Clients/src/presentation/components/RichTextEditor/index.tsx

```
@@ -0,0 +1,147 @@
+  const applyFormatting = (type: string) => {
+    if (!editor) return;
+
+    switch (type) {
+      case "bold":
+        editor.chain().focus().toggleBold().run();
+        break;
+      case "italic":
+        editor.chain().focus().toggleItalic().run();
+        break;
+      case "uppercase":
+        const uppercaseText = editor.getText().toUpperCase();
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)